src/app/components/Header/index.jsx

```diff
@@ -9,9 +9,35 @@ const StyledHeader = styled.header`
   padding: ${GEL_SPACING_DBL}px;
 `;
 
+const HiddenText = styled.span`
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+`;
+
 const Header = () => (
   <StyledHeader role="banner" aria-label="BBC News">
-    <a href="https://www.bbc.co.uk/news">BBC News</a>
+    <a href="https://www.bbc.co.uk/news">
+      <svg
```

Comment:
The required accessibility attributes, `focusable="false"` and  `aria-hidden="true"` documented in https://github.com/bbc/simorgh/issues/368 are not present on the SVG.

Reply:
This has been added.